### PR TITLE
Add editApp command and parser testing

### LINKED ISSUE #202: editApp command and parser testing

### PR DESCRIPTION
**Pull request by NecroWolf28**
Fixes #202

### CODE REVIEW

File: src/test/java/seedu/placebook/logic/commands/EditAppCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+        testSchedule = new Schedule();
+        testSchedule.addAppointment(appointment1);
+        testSchedule.addAppointment(appointment2);
+        model = new ModelManager(testContacts, new UserPrefs(), testSchedule);
+        System.out.println(model.getFilteredAppointmentList().get(0));
+        System.out.println(model.getFilteredAppointmentList().get(1));
+    }
+
+    @Test
+    public void execute_someFieldsSpecifiedUnfilteredList_success() {
```
Comment (YaleChen299) [inline src/test/java/seedu/placebook/logic/commands/EditAppCommandTest.java:67]: should there be a filteredList success testcase?

**Review by YaleChen299 — COMMENTED** [review]
Generally LGTM! Thanks for the hard work! maybe just one minor nits, not sure if it is critical

File: src/test/java/seedu/placebook/logic/commands/EditAppCommandTest.java (hunk 1 above)
Comment (NecroWolf28) [reply src/test/java/seedu/placebook/logic/commands/EditAppCommandTest.java:67]: I'll add it in, thanks!

**Review by YaleChen299 — APPROVED** [review]
LGTM

**Pull request merged**